tools/squidclient/squidclient.cc

```diff
@@ -177,6 +178,56 @@ shellUnescape(char *buf)
     *d = '\0';
 }
 
+/// [Proxy-]Authorization header producer
+class Authorization
+{
+public:
+    Authorization(const char *aHeader, const char *aDestination):
+        header(aHeader), destination(aDestination), user(nullptr), password(nullptr) {}
```

Comment:
user and password are C++11 initialized. They should not need to be repeat-initialized here.

Reply:
Fixed.